distributions/rhel-85/rhel-85.json

```diff
@@ -11,7 +11,7 @@
       "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8.5/x86_64/baseos/os",
       "rhsm": true
     }, {
-      "id": "baseos",
+      "id": "appstream",
```

Comment:
I used this was the bug? For the future, could you explain it in the commit message?

Reply:
done!